Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -584,6 +584,12 @@ def audit_homepage
       problem "Please use https:// for #{homepage}"
     end
 
+    case homepage
+    when %r{^http://([^/]*)sourceforge\.net(/|$)}i,
+         %r{^http://([^/]*)sf\.net(/|$)}i
+      problem "#{homepage} should be `https://#{$1}sourceforge.io/`"
```

Comment:
The regexes should probably be:

```ruby
%r{^http://([^/]*\.)?sourceforge\.net(/|$)}i

%r{^http://([^/]*\.)?sf\.net(/|$)}i

```

Otherwise `unrelated-sourceforge.net` and `unrelated-sf.net` would also be matched.